build.sc

```diff
@@ -1,34 +1,26 @@
 import mill._, scalalib._, scalajslib._, scalanativelib._, publish._
-import $ivy.`de.tototec::de.tobiasroeser.mill.vcs.version::0.3.1`
+import $ivy.`de.tototec::de.tobiasroeser.mill.vcs.version::0.3.1-6-e80da7`
+import $ivy.`com.github.lolgab::mill-mima::0.0.20`
 import de.tobiasroeser.mill.vcs.version.VcsVersion
-import $ivy.`com.github.lolgab::mill-mima::0.0.19`
 import com.github.lolgab.mill.mima._
-import mill.scalalib.api.Util.isScala3
+import mill.scalalib.api.ZincWorkerUtil.isScala3
 
-val dottyCommunityBuildVersion = sys.props.get("dottyVersion").toList
+val dottyCommunityBuildVersion = sys.props.get("dottyVersion")
 
-val scalaVersions =
-  "2.12.17" :: "2.13.8" :: "2.11.12" :: "3.1.3" :: dottyCommunityBuildVersion
+val scalaVersions = Seq("2.11.12", "2.12.17", "2.13.8", "3.1.3") ++ dottyCommunityBuildVersion
 
-val scalaJSVersions = scalaVersions.map((_, "1.10.1"))
-val scalaNativeVersions = scalaVersions.map((_, "0.4.5"))
-
-trait MimaCheck extends Mima {
-  def mimaPreviousVersions = VcsVersion.vcsState().lastTag.toSeq
-}
-
-trait FansiModule extends PublishModule with MimaCheck {
+trait FansiModule extends PublishModule with Mima with CrossScalaModule with PlatformScalaModule {
   def artifactName = "fansi"
 
   def publishVersion = VcsVersion.vcsState().format()
 
-  def crossScalaVersion: String
-
   // Temporary until the next version of Mima gets released with
   // https://github.com/lightbend/mima/issues/693 included in the release.
```

Comment:
@lolgab now that we've upgraded mill-mima, can we remove this>

Reply:
Yes we can. If the `mill __.mimaReportBinaryIssues` pass it means there are no problems.